config/routes.rb

```diff
@@ -13,6 +13,7 @@
       resources :responses, param: :id
       get '/response_flow_from_email', to: 'responses#response_flow_from_email'
       get '/all_emotions', to: 'emotions#all_emotions'
+      get '/see_the_results', to: 'responses#see_the_results'
```

Comment:
`/see_results` might be a better URI than `/see_the_results` since it's shorter and means the same thing.